spec/services/proceeding_type_filter_spec.rb

```diff
@@ -17,8 +17,8 @@
     context "and it has a non-SCA proceeding" do
       let(:current_proceedings) { %w[DA001] }
 
-      it "returns all proceedings minus the current one and sca_related proceedings" do
-        expect(proceeding_type_filter.count).to eq 134
+      it "returns all proceedings minus the current one, all PLF and sca_related proceedings" do
+        expect(proceeding_type_filter.count).to eq 39
```

Comment:
I wonder if there is a more declarative test that could be used. Having just a count is ok but not explicit, something like...

```
expect(proceeding_type_filter.map(&:ccms code).uniq).not_to include("^PBM")
```
This is just for illustration purposes and won't work i imagine as `include` probably does not take regex's